rancher/v1.6/en/installing-rancher/installing-server/no-internet-access/index.md

```diff
@@ -137,3 +137,21 @@ After accessing the UI, you can click on the **Add Host** button. This will imme
 The cloud providers will not work as Rancher uses `docker-machine` to provision the hosts through the cloud providers. Click on the **Custom** icon to add the host.
 
 The command from the UI can be used on any machine that has Docker configured to use HTTP proxy.
+
+### Catalogs
+On every release, the `rancher/server` container will be built containing a cached copy of the [Rancher catalog](https://github.com/rancher/rancher-catalog) and [community catalog](https://github.com/rancher/community-catalog) belonging to that release.
+
+#### Rancher catalog
+As we only support one release of [infrastructure services]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/rancher-services/) per Rancher release, you do not have to mirror or update the Rancher catalog to be supported or up-to-date.
+
+#### Community catalog
+The [community catalog](https://github.com/rancher/community-catalog) is updated regularly by the community. If you need updates to this catalog, you will need to make sure that you can either access `https://git.rancher.io/community-catalog.git` or that you configure the community catalog to be updated from your own mirrored Git repository.
+
+#### Configuring catalogs on startup
+To point your `rancher/server` container to the correct repositories upon start, you can configure the environment variable `DEFAULT_CATTLE_CATALOG_URL`. The variable `$RELEASE` will be expanded to `v1.6-release`.
+
+```
+$ sudo docker run -d --restart=unless-stopped -p 8080:8080 \
+ -e DEFAULT_CATTLE_CATALOG_URL='{"catalogs":{"community":{"url":"https://your_local_git_server/community-catalog.git","branch":"master"},"library":{"url":"https://your_local_git_server/rancher-catalog.git","branch":"${RELEASE}"}}}' \
```

Comment:
i'm not sure what this recommendation is for/doing. is it so that your library catalog could be pointed to a different location? if so, we shouldn't use `$RELEASE`, but have them create their own branches.

Reply:
Purpose is people automating the deployment of `rancher/server` with the correct catalog sources automatically. I added a section `on startup` and `manual` and changed the branch name.